l10n_it_fatturapa/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Italian Localization - Fattura elettronica - Base',
-    'version': '10.0.2.3.6',
+    'version': '10.0.2.3.7',
```

Comment:
Data l'entità della modifica, farei almeno `2.4.0`

Reply:
@sherpya c'è anche da risolvere i conflitti con un rebase

Reply:
> ```python
> fatturapa_v_1_1
> ```

l'ho  potato questo messo un __init__.py vuoto

Reply:
che poi in effetti se si mette lì il check forse si può evitare nel due autogenerati